pkgs/development/compilers/scas/default.nix

```diff
@@ -0,0 +1,22 @@
+{ fetchFromGitHub, stdenv, cmake }:
+
+
+stdenv.mkDerivation rec {
+  pname = "scas";
+  version = "0.4.6";
+  src = fetchFromGitHub {
+    owner = "KnightOS";
+    repo = "scas";
+    rev = "${version}";
+    sha256 = "1c6s9nivbwgv0f8n7j73h54ydgqw5dcpq8l752dfrnqg3kv3nn0h";
+  };
+  nativeBuildInputs = [ cmake ];
+  installTargets = [ "install" ];
```

Comment:
Is setting `installTargets` necessary? If I'm not mistaken the build script automatically uses the install target.